spec/services/search_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'search food request' do
+  it 'can return json for a search result' do
+    json = FDCService.search_foods("sweet potatoes")
+
+    expect(json).to be_a(Hash)
+    expect(json[:foods]).to be_an(Array)
+  end
```

Comment:
This testing needs to be much more robust and check for the totalResults and the correct number of food objects as well as their attributes.